internal/services/http_server/service.go

```diff
@@ -21,10 +24,14 @@ type HttpServer struct {
 	l        *slog.Logger
 }
 
-func New(cfg *Config, handlers *handlers.Handlers, l *slog.Logger) *HttpServer {
+func New(address, port string, handlers *handlers.Handlers, l *slog.Logger) *HttpServer {
+	intPort, err := strconv.Atoi(port)
+	if err != nil {
+		intPort = defaultPort
```

Comment:
As always, I'd prefer the app to exit if the inputs/configuration are invalid. So I would use `defaultPort` if `port` is missing, but exit with an error if `port` is invalid